compute/apiv1/smoke_test.go

```diff
@@ -593,3 +598,35 @@ func TestCapitalLetter(t *testing.T) {
 		t.Fatalf("got(-),want(+):\n%s", diff)
 	}
 }
+
+func TestHeaders(t *testing.T) {
+	if testing.Short() {
```

Comment:
nit: you can remove the skip. This test does not hit any backend resources so it should be fine to run all the time.

Reply:
I think the client factory method will try to load ADC & fail if not present, so if this test is using an insecure endpoint we may want to also pass `WithoutAuthentication` [option](https://pkg.go.dev/google.golang.org/api/option#WithoutAuthentication).

Reply:
Done